packages/components/inputs/rich-text-utils/src/rich-text-body/dropdown.tsx

```diff
@@ -108,7 +109,13 @@ const Dropdown = (props: TDropdownProps) => {
                         value={item.value}
                         isSelected={props.getIsItemSelected(item)}
                       >
-                        {item.label}
+                        <div
```

Comment:
**nit** perhaps we can have a private styled component for this instead of using inline CSS:
```
const DropdownItemLabelWrapper = styled.div`
  margin: ${designTokens.marginForRichTextDropdownItemLabel};
`;
```

Reply:
> AWESOME 💪
> 
> One last thing from my end: there is a tiny issue with `cursor: not-allowed` ![2023-03-15 16 41 50](https://user-images.githubusercontent.com/49066275/225363884-92fc10af-2b43-44d3-8702-7b747036ecf9.gif) [ ![2023-03-15 16 41 50](https://user-images.githubusercontent.com/49066275/225363884-92fc10af-2b43-44d3-8702-7b747036ecf9.gif) ](https://user-images.githubusercontent.com/49066275/225363884-92fc10af-2b43-44d3-8702-7b747036ecf9.gif) [ ](https://user-images.githubusercontent.com/49066275/225363884-92fc10af-2b43-44d3-8702-7b747036ecf9.gif)

Yes, this is also happening on the default behaviour theme if you zoom in and hover over the space between two editor buttons. I can do a follow up to see why this is happening.